include/bl_provision.rst

```diff
@@ -0,0 +1,22 @@
+.. _doc_bl_provision:
```

Comment:
Do we do copyright in rst?

Reply:
Not sure, but the convention seems to be 'no'.